packages/unit-vitest/src/install.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Performs a deep merge of objects and returns new object. Does not modify
+ * objects (immutable) and merges arrays via concatenation.
+ * based on https://stackoverflow.com/a/49798508
+ *
+ * @param {...object} sources - Objects to merge
+ * @returns {object} New object with merged key/values
+ */
+
+function __mergeDeep(...sources) {
+  let result = {};
+  for (const source of sources) {
+    if (source instanceof Array) {
+      if (!(result instanceof Array)) {
+        result = [];
+      }
+      result = [...result, ...source];
+    } else if (source instanceof Object) {
+      // eslint-disable-next-line prefer-const
+      for (let [key, value] of Object.entries(source)) {
+        if (value instanceof Object && key in result) {
+          value = __mergeDeep(result[key], value);
+        }
+        result = { ...result, [key]: value };
+      }
+    }
+  }
+  return result;
+}
+
+const ciCommand = 'vitest run';
+
+// make sure the object exists
+let extendPackageJson = {
+  devDependencies: {
+    '@vue/test-utils': '^2.0.0',
```

Comment:
Is is possible to provide this directly from the AE and avoid polluting devland package.json? Or will Vite complain?
Another possible problem would be that we lose autoimport for "mount" and other related helpers
Could you check?

Reply:
If not, as I expect, we need to externalize Vitest too, as we're not getting autoimport suggestions for that either
I guess it's time to abandon our mission to manage everything for the user and surrend to TS preventing us to do so

In that case, we need to add all deps into this package devDeps, then add peerDependencies with correct version restrictions
And I'm afraid we'll need to start doing this with other AEs later on 🥲

Reply:
I'm not that deep into the field of dependencies and such, I will go with your view on this, I just copied the setup of `unit-jest` as far as I know :) If I did it differently that is by accident.

Reply:
You copied correctly, the problem is into Jest AE too 🤣 
I'd go with externalizing it as the whole ecosystem is moving in that direction, but using restrictive constraints into peerDeps to keep control of the dependency version